nasa/harmony
# Harmony 1770 - EDR area routes

## Jira Issue ID
HARMONY-1770

## Description
Adds support for the OGC EDR area route for spatial subsetting via area using WKT.

## Local Test Steps
1. Rebuild services
```
npm run build-all
```
2. start harmony
```
./bin/bootstrap-harmony
./bin/start-dev-services
```
3. Execute an `area` query
```
 curl -Ln -bj "http://localhost:3000/ogc-api-edr/1.1.0/collections/C12342
08438-POCLOUD/area?maxResults=1&parameter-name=all&coords=POLYGON%20%28%28-65.390625%20-13.239945%2C%20-29.882813%20-50.958427%2C%2017.929688%2030.145127%2C%20-65.3906
25%20-13.239945%29%29" -o file.hdf
```
4. Open `file.hdf` in `panoply` and verify that it looks okay (should be a small rectangle east of South America
5. Verify that the online documentation looks okay in a browser
```
http://localhost:3000/docs
```

## PR Acceptance Checklist
